ui/static/ui/css/mit-lore.css

```diff
@@ -494,3 +494,13 @@ ul.repos {
     padding: 0px;
     margin: 0px;
 }
+
+i.add-vocabulary-spinner {
```

Comment:
Please reindent to match rest of CSS (this was merged into master just recently so you may need to rebase first)